client/state/selectors/test/has-user-asked-a-directly-question.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { hasUserAskedADirectlyQuestion } from '../';
+
+describe( 'hasUserAskedADirectlyQuestion()', () => {
+	const questionData = {
```

Comment:
You can choose to ignore this since its not a big deal :smile:

Maybe deep freeze this for an overkill test